### ISSUE TITLE
Convert components to angle bracket syntax

### ISSUE THREAD

**Issue by rwwagner90** [emberjs/ember-inspector#947]
We should be able to support using angle bracket components, in inspector itself. We need to fix the build first, but slowly converting components over would move us in the right direction.

**Comment by prasannavijayan** [comment]
Shall I take a look at this?

**Comment by nummi** [comment]
+1 to slowly moving them over. A Big Bang PR will definitely cause some errors.

**Comment by rwwagner90** [comment]
I need to cut some new branches, so we can get the build working here again, then we can begin work on this.

**Comment by akankshadharkar** [comment]
Is this still something that needs to be done? I can pick it up if so

**Comment by rwwagner90** [comment]
@akankshadharkar yes, it is something that needs to be done. We would love the help, if you have the time!

**Comment by akankshadharkar** [comment]
@rwwagner90 Awesome, I'll look over the requirements and take this up then :)

**Comment by sumitd94** [comment]
Is this still pending? I can take this up. I Will need some more info on the requirements and then I can start working on it.

**Comment by rwwagner90** [comment]
Hi @sumitd94, yes this still needs to be completed. Please feel free to convert things a component or two at a time and submit PRs! No real requirements other than converting to angle bracket syntax.

**Comment by sumitd94** [comment]
Ok Sure, I will pick this up and raise PRs. If possible could you please let me know one component which is not using the angle bracket invocation? just for reference purposes.
